ui/ui.go

```diff
@@ -81,6 +85,8 @@ func debugHandler(cfg Config) http.Handler {
 
 		cfg.DB.DbUser = "********"
 		cfg.DB.DbPassword = "********"
+		cfg.Publicip = publicIP()
+		cfg.Hostname, _ = os.Hostname()
```

Comment:
Could we add the db connection string. without password?  plus the data and or time?

Reply:
The rest of the structure contains the database details.

I'll add a time string in there though.

Reply:
https://github.com/servian/TechChallengeInstructions